terraform/cloud-platform-components/resources/fluentd-es/templates/fluentd-es-config.yaml

```diff
@@ -0,0 +1,338 @@
+#This file deploys a configMap that supports the main fluentd-es file. Apart from this comment and the line directly below, the pther comments are there by default from the original template.
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: fluentd-es-config-v0.1.4
+  namespace: {{ .Values.namespace }}
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+data:
+  system.conf: |-
+    <system>
+      root_dir /tmp/fluentd-buffers/
+    </system>
+  containers.input.conf: |-
```

Comment:
As discussed, please extract these config files like you've done for `system.input.conf`